torch/testing/_internal/opinfo/core.py

```diff
@@ -852,7 +852,12 @@ class OpInfo:
     is_factory_function: bool = False
 
     def __post_init__(self):
-        self._original_opinfo_args = asdict(self).copy()
+        try:
+            self._original_opinfo_args = asdict(self).copy()
+        except TypeError:
```

Comment:
Is there a way to do this without catching ALL TypeErrors? Easy way to accidentally suppress too much

Reply:
I can rewrite the CustomOp OpInfos to use lambdas, e.g. `lambda *args, **kwargs: CustomOp(*args, **kwargs)`. Lambdas can be deep-copied but the CustomOp object cannot